src/iDynFor/KinDynComputations.tpp

```diff
@@ -23,6 +74,11 @@ inline bool KinDynComputationsTpl<Scalar, Options, JointCollectionTpl>::loadRobo
                                                                                   m_pinModel,
                                                                                   verbose);
 
+    m_pinData = pinocchio::DataTpl<Scalar, Options, JointCollectionTpl>(m_pinModel);
+
+    // Resize m_pin_model_position to right size
+    m_pin_model_position = pinocchio::randomConfiguration(m_pinModel);
```

Comment:
Is there a ``zero`` configuration?

Reply:
Not as a dedicated function, but I manually did the same: https://github.com/ami-iit/idynfor/pull/21/commits/27f829ebf7e2bcb2daccd434a025dae88b98070e .